addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -522,6 +522,8 @@ function enforceTablesResponsivity(editable) {
         commonTable.style.height = '100%';
         const commonTr = document.createElement('tr');
         const commonTd = document.createElement('td');
+        commonTd.style.display = "flex";
+        commonTd.style.flexWrap = "wrap";
```

Comment:
I would be quite surprised if flex was supported in every email clients, especially outlook?

Reply:
I updated the fix,  but I hat to almost revert all of this [commit](https://github.com/odoo/odoo/commit/e961757c19e5052ee27e6354c021980e0d026929) , I tested in emailonacid.com and it seems to be responsive with just this change, If I missed something , let me know !

Reply:
What do you mean by responsive, do they stack ?

Reply:
Yes, they stack in small screen size, I will add some pictures for mobile view too

Reply:
UPD: I just checked and it seems to be broken in some devices , I will find another solution

Reply:
UPD: It works if I add the change in the xml ( style in head element and not in body) in all devices, but some doesn't support style element in body